pkg/download/downloader.go

```diff
@@ -33,26 +33,25 @@ import (
 // download gets a file from the internet in memory and writes it content
 // to a Verifier.
 func download(url string, verifier Verifier, fetcher Fetcher) (io.ReaderAt, int64, error) {
-	glog.V(2).Infof("Fetching %q", url)
 	body, err := fetcher.Get(url)
 	if err != nil {
-		return nil, 0, errors.Wrapf(err, "could not download %q", url)
+		return nil, 0, err
```

Comment:
removing wrapping makes me nervous. :| :|

Reply:
what if we said: `failed to obtain plugin archive`

Reply:
sounds good